bin/scripts/lib/fonts.json

```diff
@@ -887,6 +887,21 @@
       "isMonospaced": true,
       "description": "Dashed zero, curved and straight character lines"
     },
+    {
+      "unpatchedName": "Server Mono",
+      "licenseId": "OFL-1.1-no-RFN",
+      "RFN": false,
+      "version": "0.0.8",
```

Comment:
They put `1.0.0` resp. `1.000` into the font file, so that is an upstream bug :unamused: